script/countries.py

```diff
@@ -0,0 +1,32 @@
+"""Helper script to update country list.
+
+ISO does not publish a machine readable list free of charge, so the list is generated
+with help of the pycountry package.
+"""
+import black
+import pycountry
+
+BASE = """
+\"\"\"Automatically generated by currencies.py.
+
+The values are directly corresponding to the ISO 3166 standard. If you need changes
+to the political situation in the world, please contact the ISO 3166 working group.
+
+To update, run python3 -m script.currencies
+\"\"\"
+
+COUNTRIES = {}
+""".strip()
+
+countries = sorted({x.alpha_2 for x in pycountry.countries})
+
+countries_path = "homeassistant/generated/countries.py"
+with open(str(countries_path), "w") as fp:
+    fp.write(
```

Comment:
Could also use `pathlib.Path("homeassistant/generated/countries.py").write_text(...)`, but either works.

Reply:
Modified to use pathlib.